samples/02_power_users_developers/openstreetmap_exploration.ipynb

```diff
@@ -0,0 +1,1338 @@
+{
```

Comment:
<ul><li>I suggested hyperlinking to the <a href="https://developers.arcgis.com/python/api-reference/arcgis.geoenrichment.html#enrich" target="_blank"><em>enrich()</em></a> api function for those users unfamiliar with it</li><li>I think it would help to link to the<a href="https://developers.arcgis.com/rest/geoenrichment/api-reference/geoenrichment-service-overview.htm" target="_blank"> Geoenrichment Service</a> and the <a href="https://developers.arcgis.com/rest/geoenrichment/api-reference/enrich.htm" target="_blank">Enrich </a>task so users can investigate what attributes get added to the data when no variables are entered into the <a href="https://developers.arcgis.com/python/api-reference/arcgis.geoenrichment.html#enrich" target="_blank"><em>enrich()</em></a> function.</li><li>change <em>assesed </em>to <em>assessed</em></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>